Tangram not compiling on RPi

```
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp: In static member function ‘static void* Tangram::GL::mapBuffer(GLenum, GLenum)’:
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp:261:46: error: ‘glMapBufferOES’ was not declared in this scope
     auto result =  glMapBuffer(target, access);
                                              ^
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp: In static member function ‘static GLboolean Tangram::GL::unmapBuffer(GLenum)’:
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp:266:39: error: ‘glUnmapBufferOES’ was not declared in this scope
     auto result = glUnmapBuffer(target);
                                       ^
In file included from /home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp:4:0:
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp: In static member function ‘static void Tangram::GL::bindVertexArray(GLuint)’:
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp:277:37: error: ‘glBindVertexArray’ was not declared in this scope
     GL_CHECK(glBindVertexArray(array));
                                     ^
/home/patricio/paparazzi/tangram-es/core/src/gl/error.h:31:24: note: in definition of macro ‘GL_CHECK’
 #define GL_CHECK(STMT) STMT;
                        ^
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp: In static member function ‘static void Tangram::GL::deleteVertexArrays(GLsizei, const GLuint*)’:
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp:280:44: error: ‘glDeleteVertexArrays’ was not declared in this scope
     GL_CHECK(glDeleteVertexArrays(n, arrays));
                                            ^
/home/patricio/paparazzi/tangram-es/core/src/gl/error.h:31:24: note: in definition of macro ‘GL_CHECK’
 #define GL_CHECK(STMT) STMT;
                        ^
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp: In static member function ‘static void Tangram::GL::genVertexArrays(GLsizei, GLuint*)’:
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp:283:41: error: ‘glGenVertexArrays’ was not declared in this scope
     GL_CHECK(glGenVertexArrays(n, arrays));
                                         ^
/home/patricio/paparazzi/tangram-es/core/src/gl/error.h:31:24: note: in definition of macro ‘GL_CHECK’
 #define GL_CHECK(STMT) STMT;
                        ^
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp: In static member function ‘static void* Tangram::GL::mapBuffer(GLenum, GLenum)’:
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp:264:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp: In static member function ‘static GLboolean Tangram::GL::unmapBuffer(GLenum)’:
/home/patricio/paparazzi/tangram-es/core/common/platform_gl.cpp:269:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
CMakeFiles/paparazzi_worker.dir/build.make:261: recipe for target 'CMakeFiles/paparazzi_worker.dir/tangram-es/core/common/platform_gl.cpp.o' failed
```

--- a/core/common/platform_gl.h
+++ b/core/common/platform_gl.h
@@ -51,8 +51,15 @@ extern PFNGLGENVERTEXARRAYSOESPROC glGenVertexArraysOESEXT;
 #include "bcm_host.h"
 
 #include <GLES2/gl2.h>
+#include <GLES2/gl2ext.h>
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
+
+// Dummy VertexArray functions
+static void glBindVertexArray(GLuint array) {}
+static void glDeleteVertexArrays(GLsizei n, const GLuint *arrays) {}
+static void glGenVertexArrays(GLsizei n, GLuint *arrays) {}
+
 #endif
 
 #if defined(PLATFORM_ANDROID) || defined(PLATFORM_IOS) || defined(PLATFORM_RPI)

--- a/rpi/src/platform_rpi.cpp
+++ b/rpi/src/platform_rpi.cpp
@@ -128,8 +128,3 @@ void cancelUrlRequest(const std::string& _url) {
 void setCurrentThreadPriority(int priority) {}
 
 void initGLExtensions() {}
-
-// Dummy VertexArray functions
-GL_APICALL void GL_APIENTRY glBindVertexArray(GLuint array) {}
-GL_APICALL void GL_APIENTRY glDeleteVertexArrays(GLsizei n, const GLuint *arrays) {}
-GL_APICALL void GL_APIENTRY glGenVertexArrays(GLsizei n, GLuint *arrays) {}

